playbooks/foreman.yml

```diff
@@ -1,4 +1,4 @@
-- hosts: all
+- hosts: " {{ group_name | default('all') }}"
```

Comment:
Hello,

I'm thinking of updating all the plays, like this. with this change, we should be able to set the hosts during the runtime, and if we keep this empty, no problem, it will be as is at this moment.

I can submit this request for all the plays in another PR, if you believe it to be interesting.

Thank you!
Waldirio

Reply:
I don't follow entirely... could you provide an example command as a follow up comment?

Reply:
Hello @ehelms 

Sure, in the forklift example, we can see the command below
```
echo -e "[devel]\ntest.example.com" > inventories/local_inventory
```

This will include the `test.example.com` in the local_inventory, and in the `ansible.cfg`, we are reading pointing inventory to the directory `inventory = $PWD/inventories/`, which means, this will read all the files that we have created.

Assuming that I have already my devel env pointing to `test.example.com` and I would like to create a second server, now, using katello, for instance. I should be able to create a second entry on the `inventories/local_inventory` file and point the installation just for that server.

```
echo -e "[katello]\nnewserver.example.com" >> inventories/local_inventory
```

Now, the new group will be `katello`, but with the current configuration, once I call the play, it will be executed against all the servers on this inventory file/folder. With this change, we can run against all the servers, if we omit the extra vars, or we can act against that specific group, in this case, `katello`.

Basically, if we pass the group_name as `-e group_name=katello`, this will call only that group, if we leave it, this will use the standard wich will be all servers in the inventory.

Please, let me know if it's clear now.

Thank you!
Waldirio

Reply:
I thought the same could be achieved through native ansible method:

```
ansible-playbook playbooks/foreman.yml -l host1
ansible-playbook playbooks/foreman.yml -l katello
```

https://docs.ansible.com/ansible/latest/inventory_guide/intro_patterns.html#patterns-and-ansible-playbook-flags

Reply:
@ehelms that's perfect! Removing this additional work!

Thank you!